docs/getting_started_guide/Installation_Options/r10k/Installing_SIMP_using_r10k.rst

```diff
@@ -152,7 +155,7 @@ times in order to get the new system in order.
       }
 
   SIMP also moves the location of the ssh authorized_keys file to
-  ``/etc/ssh/local_keys/%u``, so copy it there before logging out.
+  ``/etc/ssh/local_keys/%u``, so make sure to copy it there before logging out.
```

Comment:
Or do we want to tell the user how to configure back to the normal location?

Reply:
Nope